browser/css/notebookbar.css

```diff
@@ -764,3 +764,12 @@ html[data-theme='dark'] .savemodified.unotoolbutton .unobutton img {
 	width: var(--btn-img-size);
 	height: var(--btn-img-size);
 }
+
+#masterpageall_icons .notebookbar.ui-iconview-entry {
+	height: 2.75rem;
+}
+
+#masterpageall_icons .notebookbar.ui-iconview-entry img {
```

Comment:
I'm not sure if all have the same aspect ratio, we can have 4:3 or 16:9 slides.
But possibly it doesn't make much difference in such small preview where you want to see approximate effect of applying it.